src/client/components/common/Card/CardElement/index.tsx

```diff
@@ -29,27 +29,36 @@ const SortIcon = styled(({ activeSort, sortType, ...props }: SortIconProps) => <
   ${({ activeSort, sortType, theme }) =>
     activeSort &&
     `
-    color: ${theme.colors.onSurfaceH2};
+    color: ${theme.colors.titles};
     transform: ${sortType === 'asc' ? 'rotateZ(180deg)' : 'rotateZ(0deg)'};
   `}
 `;
 
-const Header = styled.div<{ onClick?: () => void }>`
+const Header = styled.h3<{ onClick?: () => void }>`
```

Comment:
Is this using the new yearn-web-lib? If so, we should have a Typography for h3

Reply:
Nope, not being used. Also I would not put an h3 on card element, since an element can have another elements inside. I discussed with pentacle about it today since:
![telegram-cloud-photo-size-1-5093984801542744590-y](https://user-images.githubusercontent.com/78362532/163130968-d5d0f7a1-e470-4a70-9037-9c192f05bf63.jpg)